src/payment/checkout/payment-form/flex-microform/FlexMicroform.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { setPaymentTokenFetch } from '../../../payment-methods/cybersource/service';
+
+class FlexMicroform extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            microform: null,
+        }
+        this.onLoad = this.onLoad.bind(this);
+        this.getPaymentToken = this.getPaymentToken.bind(this);
+    }
+
+    componentDidMount() {
+        setPaymentTokenFetch(this.getPaymentToken);
+
+        const script = document.createElement('script');
```

Comment:
One concern here is that we're adding this script every time this component mounts.  Any component above this in the component tree could unmount/re-mount this component at any time, in which case we would re-attach the script to the body, which could have pretty undefined results.  Seems like we'd want to attach this script at a more predictable point in the application lifecycle.  Sometime that we know only happens once.

I think the right thing to do here would be to attach it in the API layer when the basket is returned and keep track of whether it's been added.  That could be done by checking if `window.Flex` exists before appending the script, and if it doesn't, appending it.

Another very different option would be to conditionally add the script to the index.html file via an environment variable.  This would, of course, require coordinating the ecommerce basket's `flexMicroformEnabled` flag with the config of this MFE.  That may not be desirable.